interpreter/CMakeLists.txt

```diff
@@ -429,6 +442,10 @@ Please install Python or specify the PYTHON_EXECUTABLE CMake variable.")
 
   # We are in the case of NOT builtin_llvm
   if (builtin_clang)
+    # remove clang-cpp from CLANG_LINKS_TO_CREATE to avoid clashes with
+    # install-clang-cpp target defined by LLVM's cmake module
+    set(CLANG_LINKS_TO_CREATE clang++ clang-cl)
```

Comment:
I was wondering on which systems that avoids clashes?

Reply:
On Nix/nixpkgs. You can reproduce the error message with
```
nix-build \
  -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/dd98b100651cfbb8804f32d852f75ef7c97a6b74.tar.gz \
  -E 'with import <nixpkgs> {}; root.overrideAttrs (old: { cmakeFlags = old.cmakeFlags ++ [ "-Dbuiltin_llvm=OFF" ]; buildInputs = old.buildInputs ++ [ llvm_9 ]; })'
```